### DISCUSSION TITLE
Important Announcement: Seeking Your Ideas for New Features in CryptoChat!

### DISCUSSION THREAD

**Discussion by HalilDeniz** [HalilDeniz/CryptoChat#5] [Q&A] [answered]
Hello CryptoChatCommunity,

I'm excited to hear your thoughts and suggestions for the future of CryptoChat! We aim to enhance the user experience further, and I need your input on two new features I plan to implement.

### 📁 **New Feature: File Sending**
- Users will soon be able to send files with the `/sendfile <everyone/username> <filename>` command. The sent file will automatically be saved on the recipient's device. How can I make this feature more functional? What security measures should we consider?

### 🚀 **Performance Improvement: Large Data Transfer**
- I am considering splitting data into chunks for more efficient large data transfers. What technical challenges might we encounter? How could this affect the user experience?

Your feedback is crucial for the development of CryptoChat. Please share your thoughts, suggestions, and any concerns you might have. We are open to all ideas and want to hear the voice of our community!

### 📬 **How Can You Provide Feedback?**
- You can leave your comments below this announcement or reach us directly via our [GitHub Issue page](https://github.com/HalilDeniz/CryptoChat/issues).

### 🤝 Contributing to CryptChat
- We deeply value the contributions of our community and are excited to see CryptChat grow and evolve through your collaboration. Whether it's adding new features, fixing bugs, or improving documentation, your help is always appreciated!

1. Fork the repository.
2. Create a new branch for your feature or bug fix.
3. Make your changes and commit them.
4. Push your changes to your forked repository.
5. Open a pull request in the main repository.

We look forward to working with you for a better CryptChat!

Best regards,
The CryptChat Development Team

**Comment by Tenzzyth** [comment, marked as answer]
great ideas

**Reply by HalilDeniz** [reply]
thank you so much
